src/models/components/ReasonsRadioGroup.ts

```diff
@@ -0,0 +1,15 @@
+import { getEnabledAppealReasons } from 'app/utils/FeatureChecker';
+
+export function createReasonsRadioGroup(): {}[] {
+    const enabledReasons = getEnabledAppealReasons();
+    const allReasons = [
+        { value: 'illness', text: 'Illness and health issues' },
+        { value: 'other', text: 'I\'m appealing for another reason' }
```

Comment:
Use apostrophe instead of `\'`

Reply:
I'm unsure what you mean here. An apostrophe would terminate the string prematurely without the escape character. Do you mean I should swap the enclosing apostrophes for quotation marks?

Reply:
No, I mean proper apostrophe character (`’`) that you get from Option + Shift + ] combination.

Reply:
Oh, that's a backtick, but okay. This has been refactored into the template now anyway. Marking as resolved.